db/schema.rb

```diff
@@ -399,6 +404,7 @@
     t.boolean "orders_devices"
     t.datetime "techsource_account_confirmed_at"
     t.datetime "deleted_at"
+    t.boolean "rb_level_access", default: false, null: false
```

Comment:
change was introduced elsewhere can we delete this line please

Reply:
Good spot but that one should be in main (and have just rebased)